collector_client_grpc.go

```diff
@@ -114,6 +115,8 @@ func (client *grpcCollectorClient) Report(ctx context.Context, req reportRequest
 	if req.protoRequest == nil {
 		return nil, fmt.Errorf("protoRequest cannot be null")
 	}
+
+	ctx = grpcmeta.NewOutgoingContext(ctx, grpcmeta.Pairs("LightStep-Access-Token", client.accessToken))
```

Comment:
I think this string value should be defined as a constant. I don't have strong feelings about whether there should be one per file or if we should put the header in a shared place like `collector_client.go`

Reply:
Actually, I think we should use `CanonicalHeaderKey()` like the values in collector_client_http.go

Reply:
makes sense